src/modules/ROOT/pages/changelog/202303/03.json

```diff
@@ -0,0 +1,53 @@
+{
+  "meta": {
+    "releasedOn": "2023.05.30"
+  },
+  "added": [
+    {
+      "what": [
+        "a stale data error in case a workflow was updated whilst a different user is editing the old version of the same workflow."
```

Comment:
This commit added stale data error for most resources, not just workflows & triggers.